### PR TITLE
Update french translation

### PR DESCRIPTION
**Pull request by sparunakian**
Hi,

This pull request updates the french translation and fixes a typo.

Regards,

### CODE REVIEW

**Comment by sparunakian** [comment]
Checks fail with message "demandes" is a misspelling of "demands", but "demandes" is the french translation for "demands". So these errors should be ignored.

**Comment by akiko-pusu** [comment]
Hi, @sparunakian 
I'm deeply sorry for my delayed reply.  And thank you so much for your pull request!

> but "demandes" is the french translation for "demands". So these errors should be ignored.

Ok, I'll accept this and marked as resolved.
Soon I'm going to merge into master branch!

File: config/locales/fr.yml [hunk 1]
```diff
@@ -10,16 +10,19 @@ fr:
...
   project_module_banner: "Bannière"
-  page_overview_only: "Prévisualiser seulement"
+  page_overview_only: "Aperçu seulement"
   page_all: "Sur toutes les pages"
-  page_overview_and_issues: "Conception et questions"
-  page_new_issue_only: "Nouvelle question seulement"
-  more_info: "[Plus d'Info]"
-  label_more_info: "Lien vers plus d'information."
-    
+  page_overview_and_issues: "Aperçus et demandes"
```
Comment (akiko-pusu) [inline config/locales/fr.yml]: > __[Misspell]__ "demandes" is a misspelling of "demands" ([see on Sider](https://sider.review/gh/repos/2719993/pulls/116#issue-4606754))
>
> <details>
>
> | Rule |
> | - |
> | "demandes" is a misspelling of "demands" |
>
> </details>

In French.

File: config/locales/fr.yml [hunk 2]
```diff
@@ -10,16 +10,19 @@ fr:
...
+  page_new_issue_only: "Nouvelles demandes seulement"
```
Comment (akiko-pusu) [inline config/locales/fr.yml]: > __[Misspell]__ "demandes" is a misspelling of "demands" ([see on Sider](https://sider.review/gh/repos/2719993/pulls/116#issue-4606755))
>
> <details>
>
> | Rule |
> | - |
> | "demandes" is a misspelling of "demands" |
>
> </details>

In French.

**Comment by akiko-pusu** [comment]
@sparunakian Thank you so much for your pull request!
I've just applied it to dev and master branch!

**Comment by sparunakian** [comment]
Thank you!

**Pull request merged**